test/cea/dtvcc_packet_unit.js

```diff
@@ -0,0 +1,64 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('DtvccPacket', () => {
+  /** @type {!shaka.cea.DtvccPacket} */
+  let dtvccPacket;
+
+  beforeEach(() => {
+
+  });
```

Comment:
I think you could drop this.  :-)

Reply:
oops, I should have been more attentive while copy/pasting 😳